newIDE/app/src/MainFrame/UseMultiplayerLobbyConfigurator.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+import * as React from 'react';
+import AuthenticatedUserContext from '../Profile/AuthenticatedUserContext';
+import { duplicateLobbyConfiguration } from '../Utils/GDevelopServices/Play';
+
+const gd = global.gd;
+
+type UseMultiplayerLobbyConfiguratorOutput = {
+  configureMultiplayerLobbiesIfNeeded: (
+    project: gdProject,
+    sourceGameId: string
+  ) => Promise<void>,
+};
+
+/**
+ * Hook allowing to duplicate lobby configuration from another project, useful after
+ * opening a project from an example.
+ */
+export const useMultiplayerLobbyConfigurator = (): UseMultiplayerLobbyConfiguratorOutput => {
+  const { profile, getAuthorizationHeader } = React.useContext(
+    AuthenticatedUserContext
+  );
+  const configureMultiplayerLobbiesIfNeeded = React.useCallback(
+    async (project: gdProject, sourceGameId: string) => {
+      if (!profile) {
+        console.warn(
```

Comment:
I'm not sure we need a warn here, if a user creates a project from an example without having created an account it shouldn't be a warn, just an early return?

Reply:
Oh yes sorry I'll do it differently